### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by deepsmokers**

### CODE REVIEW

File: docs/source/adaptive_learning.rst [hunk 1]
```diff
@@ -45,16 +48,43 @@ If you prefer to work with the curl, you need to follow these two stages to uplo
...
       curl -X GET "https://api.neuralet.io/api/v1/file/upload/" -H  "accept: application/json" -H "Authorization: Bearer TOKEN"
-      
-  After running this command, it will return a json containing two items. The first key is :code:`name`, which has your unique id (:code:`UUID`), and you should keep it for the config step. The second key is :code:`upload_link` that you should use as :code:`UploadURL` in the next part.
+  
+  .. note::
+    This endpoint has 2 more optional parameters:
+
+    1. label: Label indicates that in the future you can distinct the uploads from each other (in current situation only distinction between uploads is their ids which can be difficult to read).
+    
+    2. video_names (comma-separated names): Video names included in zip file in comma-separated format (like :code:`video1.mp4,video2.mp4`).
```
Comment (alpha-carinae29) [inline docs/source/adaptive_learning.rst]: What is the intent of this parameter?

File: docs/source/adaptive_learning.rst [hunk 2]
```diff
@@ -45,16 +48,43 @@ If you prefer to work with the curl, you need to follow these two stages to uplo
...
+
+    If you intend to pass these 2 parameters your url will be like this: ::
+        
+        https://api.neuralet.io/api/v1/file/upload/?label=crowded-street&?video_names=elm_street.mp4,helm_street.mp4
```
Comment (alpha-carinae29) [inline docs/source/adaptive_learning.rst]: I would add the full command so user can copy paste it.

File: docs/source/adaptive_learning.rst [hunk 3]
```diff
@@ -45,16 +48,43 @@ If you prefer to work with the curl, you need to follow these two stages to uplo
...
-  After running this command, it will return a json containing two items. The first key is :code:`name`, which has your unique id (:code:`UUID`), and you should keep it for the config step. The second key is :code:`upload_link` that you should use as :code:`UploadURL` in the next part.
+  
+  .. note::
+    This endpoint has 2 more optional parameters:
+
+    1. label: Label indicates that in the future you can distinct the uploads from each other (in current situation only distinction between uploads is their ids which can be difficult to read).
+    
+    2. video_names (comma-separated names): Video names included in zip file in comma-separated format (like :code:`video1.mp4,video2.mp4`).
+
+    If you intend to pass these 2 parameters your url will be like this: ::
```
Comment (alpha-carinae29) [inline docs/source/adaptive_learning.rst:59]: Can you add the python comment too?

File: docs/source/adaptive_learning.rst [hunk 4]
```diff
@@ -45,16 +48,43 @@ If you prefer to work with the curl, you need to follow these two stages to uplo
...
+        
+        https://api.neuralet.io/api/v1/file/upload/?label=crowded-street&?video_names=elm_street.mp4,helm_street.mp4
+
+
+
+After running this command, it will return a json containing two items. The first key is :code:`name`, which has your unique id (:code:`UUID`), and you should keep it for the config step. The second key is :code:`upload_object` which contains 2 keys: :code:`url` and :code:`fields`. 
+
+You can see the result in section below: ::
+    
+    "upload_object":{
```
Comment (alpha-carinae29) [inline docs/source/adaptive_learning.rst:69]: does the user needs to save all of these fields for next steps? If so please mention it.

File: docs/source/adaptive_learning.rst [hunk 5]
```diff
@@ -45,16 +48,43 @@ If you prefer to work with the curl, you need to follow these two stages to uplo
...
+            "AWSAccessKeyId":"ASIA...",
+            "x-amz-security-token":"IQoJb3...",
+            "policy":"eyJleHBpcmF0aW9uI...",
+            "signature":"M3bcjKw..."
+        }
+    }
 
 * Upload File:
-  In this stage, you must put the :code:`UploadLink` you have copied in the last part in place of UploadURL. For the :code:`FILE_PATH`, you need to input the path to the input.zip file you have created in the first step. ::
+  In this stage, you must put the :code:`upload_object.url` you have copied in the last part in place of UploadURL. For the :code:`FILE_PATH`, you need to input the path to the input.zip file you have created in the first step. ::
```
Comment (alpha-carinae29) [inline docs/source/adaptive_learning.rst]: shouldn't other fields change too?

File: docs/source/adaptive_learning.rst [hunk 6]
```diff
@@ -112,7 +144,7 @@ You only need to repeat the previous step and copy-paste your token in the :code
 
     curl -X POST "https://api.neuralet.io/api/v1/model/status/" -H  "accept: application/json" -H  "Content-Type: application/json" -H "Authorization: Bearer TOKEN" -d "{\"job_id\":\"JOB_ID\"}"
 
-By running the command and sending your request to our API, you may get one of the following messages for either the Teacher or Student models each time you request for the status:
+By running the command and sending your request to our API, you may get one of the following messages for either the Teacher or Student models each time you request for the status (Overall Status):
```
Comment (alpha-carinae29) [inline docs/source/adaptive_learning.rst:147]: In python client I get duplicated results, something like this:

```
{'status': 'Training (103)',
 'student_progress': '0',
 'student_status': 'Not Reached Yet (100)',
 'teacher_progress': '0',
 'teacher_status': 'Allocating Resource (101)'}
{'status': 'Training (103)', 'teacher_status': 'Allocating Resource (101)', 'student_status': 'Not Reached Yet (100)', 'teacher_progress': '0', 'student_progress': '0'}
```

File: docs/source/adaptive_learning.rst [hunk 7]
```diff
@@ -190,3 +225,140 @@ The importance of this file is for exporting your model to the EdgeTPU. In this
...
+
+Enter the address to your token text file. respectively, in the provided :code:`TOKEN_PATH` field of the command and run it. ::
+    
+    python3 client.py --token TOKEN_PATH user_jobs
+
+*Curl Command:*
+
+You only need to repeat the previous step and copy-paste your token in the :code:`TOKEN` field. ::
+
+    curl "https://api.neuralet.io/api/v1/users/me/jobs/" -H "Authorization: Bearer TOKEN"
```
Comment (alpha-carinae29) [inline docs/source/adaptive_learning.rst]: I get nothing when I run this.

File: docs/source/adaptive_learning.rst [hunk 8]
```diff
@@ -190,3 +225,140 @@ The importance of this file is for exporting your model to the EdgeTPU. In this
...
+
+Enter the address to your token text file. respectively, in the provided :code:`TOKEN_PATH` field of the command and run it. ::
+    
+    python3 client.py --token TOKEN_PATH user_uploads
+
+*Curl Command:*
+
+You only need to repeat the previous step and copy-paste your token in the :code:`TOKEN` field. ::
+
+    curl "https://api.neuralet.io/api/v1/users/me/uploads/" -H "Authorization: Bearer TOKEN"
```
Comment (alpha-carinae29) [inline docs/source/adaptive_learning.rst]: I get nothing when I run this.

File: docs/source/adaptive_learning.rst [hunk 9]
```diff
@@ -190,3 +225,140 @@ The importance of this file is for exporting your model to the EdgeTPU. In this
...
+
+Enter the address to your token text file. respectively, in the provided :code:`TOKEN_PATH` field of the command and run it. ::
+    
+    python3 client.py --token TOKEN_PATH user_detail
+
+*Curl Command:*
+
+You only need to repeat the previous step and copy-paste your token in the :code:`TOKEN` field. ::
+
+    curl "https://api.neuralet.io/api/v1/users/me/detail/" -H "Authorization: Bearer TOKEN"
```
Comment (alpha-carinae29) [inline docs/source/adaptive_learning.rst]: I get nothing when I run this.

File: docs/source/adaptive_learning.rst [hunk 10]
```diff
@@ -190,3 +225,140 @@ The importance of this file is for exporting your model to the EdgeTPU. In this
...
+**User Jobs**
+-------------
+
+Get User jobs list.
+
+*Client Code:*
+
+Enter the address to your token text file. respectively, in the provided :code:`TOKEN_PATH` field of the command and run it. ::
+    
+    python3 client.py --token TOKEN_PATH user_jobs
```
Comment (alpha-carinae29) [inline docs/source/adaptive_learning.rst]: please add `--page` as an optional parameter.

File: docs/source/adaptive_learning.rst [hunk 11]
```diff
@@ -190,3 +225,140 @@ The importance of this file is for exporting your model to the EdgeTPU. In this
...
+**User Uploads**
+----------------
+
+Get User uploads list.
+
+*Client Code:*
+
+Enter the address to your token text file. respectively, in the provided :code:`TOKEN_PATH` field of the command and run it. ::
+    
+    python3 client.py --token TOKEN_PATH user_uploads
```
Comment (alpha-carinae29) [inline docs/source/adaptive_learning.rst]: please add `--page` as an optional parameter.

**Review by alpha-carinae29 — COMMENTED** [review]
Thank you @deepsmokers for updating documentation, I tried to test an end-to-end pipeline by following this instruction. Must of this works fine, however I left some comments on ambiguous parts.

File: docs/source/adaptive_learning.rst (hunk 1 above)
Comment (deepsmokers) [reply docs/source/adaptive_learning.rst]: which one?

File: docs/source/adaptive_learning.rst (hunk 5 above)
Comment (deepsmokers) [reply docs/source/adaptive_learning.rst]: what other fields?

File: docs/source/adaptive_learning.rst (hunk 1 above)
Comment (alpha-carinae29) [reply docs/source/adaptive_learning.rst]: `video_names`

Comment (alpha-carinae29) [reply docs/source/adaptive_learning.rst]: OK I got it, you mentioned that it is for website usage.

File: docs/source/adaptive_learning.rst (hunk 5 above)
Comment (alpha-carinae29) [reply docs/source/adaptive_learning.rst]: It is clear now

File: docs/source/adaptive_learning.rst [hunk 12]
```diff
@@ -45,16 +48,43 @@ If you prefer to work with the curl, you need to follow these two stages to uplo
...
+            "signature":"M3bcjKw..."
+        }
+    }
 
 * Upload File:
-  In this stage, you must put the :code:`UploadLink` you have copied in the last part in place of UploadURL. For the :code:`FILE_PATH`, you need to input the path to the input.zip file you have created in the first step. ::
+  In this stage, you must put the :code:`upload_object.url` you have copied in the last part in place of UploadURL. Use data in :code:`upload_object.fields` from last part to pass as form-data. For the :code:`FILE_PATH`, you need to input the path to the input.zip file you have created in the first step. ::
 
-      curl "UploadURL" --upload-file FILE_PATH
+      curl "UploadURL" -F "key=UUID/input.zip" -F "AWSAccessKeyId"=ASIA..." -F "x-amz-security-token=IQoJb3..." -F "policy=eyJleHBpcmF0aW9uI..." -F "signature=M3bcjKw..." -F "file=@FILE_PATH"
```
Comment (alpha-carinae29) [inline docs/source/adaptive_learning.rst]: it seems there is an extra semicolon.

File: docs/source/adaptive_learning.rst [hunk 13]
```diff
@@ -190,3 +225,141 @@ The importance of this file is for exporting your model to the EdgeTPU. In this
...
+
+Enter the address to your token text file. respectively, in the provided :code:`TOKEN_PATH` field of the command and run it. ::
+    
+    python3 client.py --token TOKEN_PATH user_uploads --page 1
+
+*Curl Command:*
+
+You only need to repeat the previous step and copy-paste your token in the :code:`TOKEN` field. ::
+
+    curl "https://api.neuralet.io/api/v1/users/me/uploads" -H "Authorization: Bearer TOKEN"
```
Comment (alpha-carinae29) [inline docs/source/adaptive_learning.rst]: please add `page` parameter

File: docs/source/adaptive_learning.rst [hunk 14]
```diff
@@ -190,3 +225,141 @@ The importance of this file is for exporting your model to the EdgeTPU. In this
...
+
+Enter the address to your token text file. respectively, in the provided :code:`TOKEN_PATH` field of the command and run it. ::
+    
+    python3 client.py --token TOKEN_PATH user_jobs --page 1
+
+*Curl Command:*
+
+You only need to repeat the previous step and copy-paste your token in the :code:`TOKEN` field. ::
+
+    curl "https://api.neuralet.io/api/v1/users/me/jobs" -H "Authorization: Bearer TOKEN"
```
Comment (alpha-carinae29) [inline docs/source/adaptive_learning.rst]: please add `page` parameter

**Comment by deepsmokers** [comment]
@alpha-carinae29 changes has been made

**Review by alpha-carinae29 — APPROVED** [review]
Thanks @deepsmokers .
@mhejrati LGTM

**Pull request merged**